layouts/blog/post.html

```diff
@@ -20,7 +20,7 @@ <h1><a href="{{.Page.Permalink}}">{{ .Title }}</a></h1>
             {{ end }}
             </p>
         </aside>
-        <div class="post-content">
+        <div class="post-content table-wrapper">
```

Comment:
I don't think we need the `table-wrapper` class

Reply:
I observed that in the components reference section, there is a section where table-wrapper is used for controlling the overflow of the table. However, in the blog section, for the post the table doesn't contain the table wrapper class hence, the overflow of tables causes the hindrance.